lib/Onnxifi/Base.cpp

```diff
@@ -22,17 +22,22 @@
 
 namespace glow {
 namespace onnxifi {
+namespace {
+const std::string inferenceFunctionName = "inference";
```

Comment:
Why is this std::string and not "const char*" ?

Reply:
No reason, I'll change this and the below std::string